### ISSUE TITLE
Target netstandard2.0 to run with Cake.CoreCLR

### ISSUE THREAD

**Issue by git-man** [cake-contrib/Cake_Git#60]
It would be cool, if the Cake.CoreCLR compatibility could be added, as stated here: 
https://cakebuild.net/blog/2018/02/cake-v0.26.0-released 
Due to this only "netstandard2.0" needs to be targeted by the addin.

The error, which appeared to me relating to this issue was a known one: "Error: Failed to install addin 'Cake.Git'."
This misleading message lead me to: https://github.com/cake-build/cake/issues/2011 ... but that's another thing...

Thanks for your support and your great work!

**Comment by git-man** [comment]
I managed to build it on my own by only changing the csproj (please see below). Maybe you could incorporate that and create a bundle? Would be cool. Thanks for all!

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
    <ProjectGuid>{08F0A5C3-0E9C-451D-B003-14FF73699BE1}</ProjectGuid>
    <OutputType>Library</OutputType>
    <AppDesignerFolder>Properties</AppDesignerFolder>
    <RootNamespace>Cake.Git</RootNamespace>
    <AssemblyName>Cake.Git</AssemblyName>
    <TargetFramework>netstandard2.0</TargetFramework>
    <FileAlignment>512</FileAlignment>
    <NuGetPackageImportStamp>
    </NuGetPackageImportStamp>
    <TargetFrameworkProfile />
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
    <DebugSymbols>true</DebugSymbols>
    <DebugType>full</DebugType>
    <Optimize>false</Optimize>
    <OutputPath>bin\Debug\</OutputPath>
    <DefineConstants>DEBUG;TRACE</DefineConstants>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
    <DocumentationFile>bin\Debug\Cake.Git.XML</DocumentationFile>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
    <DebugType>pdbonly</DebugType>
    <Optimize>true</Optimize>
    <OutputPath>bin\Release\</OutputPath>
    <DefineConstants>TRACE</DefineConstants>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
    <DocumentationFile>bin\Release\Cake.Git.XML</DocumentationFile>
  </PropertyGroup>
  <ItemGroup>
	 <PackageReference Include="Cake.Core" Version="0.26.1" />
	 <PackageReference Include="LibGit2Sharp" Version="0.25.0-preview-0081" />
  </ItemGroup>
  <ItemGroup>
    <None Include="..\..\LICENSE.md">
      <Link>LICENSE.md</Link>
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
  </ItemGroup>
  <ItemGroup />
</Project>
```

**Comment by kentcb** [comment]
Arrived here for the same reason. Would it be possible to get some confirmation from the project admin as to whether this is likely to be incorporated soon?

**Comment by devlead** [comment]
Fixed in just released 0.19.0
